tests/console/orphaned_packages_check.pm

```diff
@@ -55,21 +55,21 @@ sub compare_orphans_lists {
 }
 
 sub run {
-    select_console 'root-console';
+    select_serial_terminal;
 
     record_info((is_offline_upgrade_or_livecd) ? 'Upgrade/LiveCD' : 'No upgrade/LiveCD', 'Upgraded or installed from LIVECD can possibly cause orphans');
 
     zypper_call('in curl') if (script_run('rpm -qi curl') == 1);
     # Orphans are also expected in JeOS without SDK module (jeos-firstboot, jeos-license and live-langset-data)
     # Save the orphaned packages list to one log file and upload the log, so QA can use this log to report bug
     # Filter out zypper warning messages and release or skelcd packages
-
-    script_run(
-q[zypper --quiet packages --orphaned | awk -F'|' 'intable && NR>2 {print $3} /^-/ { intable=1 }' | grep -v '\(release-DVD\|release-dvd\|openSUSE-release\|skelcd\)' > /tmp/orphaned.log],
+    my @orphans = split(
+        '\n',
+        script_output
```

Comment:
IMO this is too much nesting.

Reply:
You are correct!

Reply:
Then change it?

Reply:
I did, but I think it is still too much!! let me try to do more

Reply:
@Vogtinator https://openqa.opensuse.org/tests/3199317#details